lib/Backend/Chakra.Backend.vcxproj.filters

```diff
@@ -130,6 +130,8 @@
     <ClCompile Include="$(MSBuildThisFileDirectory)GlobOptBlockData.cpp" />
     <ClCompile Include="$(MSBuildThisFileDirectory)ValueInfo.cpp" />
     <ClCompile Include="$(MSBuildThisFileDirectory)JITThunkEmitter.cpp" />
+    <ClCompile Include="$(MSBuildThisFileDirectory)JavascriptNativeOperators.cpp" />
+    <ClCompile Include="EquivalentTypeSet.cpp" />
```

Comment:
Hmmm.. I don't think the *.filters needs it.  The *.filters is on used by the VS project system to put stuff in "folders".    It doesn't affect the build.  And the only reason we need $(MSBuildThisFileDirectory) is because of the windows build.

Anyway, I will add it just to be consistent.